### ISSUE TITLE
Sqlite3 #define fails to translate to zig

### ISSUE THREAD

**Issue by donaldcallen** [ziglang/zig#4964]
Running zig translate-c on /usr/include/sqlite3.h (Slackware Linux current, zig-linux-x86_64-0.5.0+701c03d08.tar.xz installed) fails to translate
#define SQLITE_STATIC      ((sqlite3_destructor_type)0)

The corresponding place in the output file:
pub const SQLITE_STATIC = @compileError("unable to translate C expr: expected ')'' here");

**Comment by Vexu** [comment]
This will be closed by #4688.

**Comment by Vexu** [comment]
#4688 landed and this now translates to 
```zig
pub const SQLITE_STATIC = @intToPtr(sqlite3_destructor_type, 0);
```
